addon/services/scheduler.js

```diff
@@ -164,3 +165,34 @@ export default Service.extend({
     run.cancel(this._nextPaintTimeout);
   }
 });
+
+if (DEBUG) {
```

Comment:
It seems weird to have this conditionally added to the service. Is this conventional, @rwjblue?

Reply:
it's from his blog here :)
 http://rwjblue.com/2017/05/01/enabling-debug-code-stripping/

Reply:
HA, I guess that answers _that_ question.